ShiftCipher.js

```diff
@@ -0,0 +1,68 @@
+// A shift cipher takes a plain text message and shifts each letter forward in the alphabet by a given number. For example, a shift cipher with a shift of 1 would turn the string 'hello' to 'ifmmp'.
+
+// Create a class ShiftCipher that takes the numerical value of the shift as a constructor parameter. The class should have two methods:
+
+// encrypt: takes a plain text string and returns a capitalized string with each letter shifted forward in the alphabet based on the set shift value.
+// decrypt: takes an encrypted message and returns a lower case string with each letter shifted back in the alphabet based on the set shift value.
+// In both methods, any character outside the alphabet should remain the same.
+// But if a character is shifted outside the alphabet in either direction it should be wrapped around to the other side. For example, encrypting a y with a shift of 4 results in C and decrypting an A with a shift of 1 result in z.
+// Example:
+
+// const cipher = new ShiftCipher(2);
+// cipher.encrypt('I love to code!'); // returns 'K NQXG VQ EQFG!'
+// cipher.decrypt('K <3 OA RWRRA'); // returns 'i <3 my puppy'
+// Feel free to reference the Unicode Table as well as the JavaScript String methods including: toUpperCase(), toLowerCase(), charCodeAt() and fromCharCode()
+
+class ShiftCipher {
+	constructor(shift) {
+		this._shift = shift;
+		this._cipherArray = [];
```

Comment:
Waarom de _chipherArray als object property definiëren? Je gebruikt hem namelijk enkel binnen de encrypt en decrypt methods zonder dat je ooit info er uit haalt in een andere method. Je kan deze wat mij betreft weg laten en gewoon als variabele met de scope van de method zelf gebruiken.

Reply:
Dit zou tevens voorkomen dat ik de array moet wissen aan het begin van de methods. Probleem was nu dat er nog data in de array stond als je na een encrypt een decrypt doet.

Reply:
Ja nu wil je bij elke method wel een lege array definiëren welke je gaat vullen. Hi nu me meer om de scope van die array.